persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/constants/BalSyntaxConstants.java

```diff
@@ -51,9 +51,9 @@ private InheritedTypeReferenceConstants() {
     public static final String ENUM = "ENUM";
     public static final String PERSIST_ERROR = "persist:Error";
     public static final String GET_G_SHEET_PERSIST_CLIENT = "googleSheetsClient = self.persistClients.get(%s);";
-    public static final String G_SHEET_CLIENT_DECLARATION = "persist:GoogleSheetsClient googleSheetsClient;";
+    public static final String G_SHEET_CLIENT_DECLARATION = "googlesheets:GoogleSheetsClient googleSheetsClient;";
```

Comment:
you can use the alias `psheets`

Reply:
Yes, but it doesn't cause any conflict. If changing this, we need to update `in-memory` with an alias to keep it identical. Isn't it?

@daneshk Shall I update all the datastore's import with an alias?

Reply:
yes, let's keep it as `googlesheets`. it aligns with the data store name as well.

Reply:
We would have just `Client` instead of `GoogleSheetsClient` let's think later